### DISCUSSION TITLE
Does the Battery Meter work for anyone?

### DISCUSSION THREAD

**Discussion by DerryLin-Tamu** [SimDynamX/SpaceTeamsPRO#79] [2024 Lunar SAR Competition Q&A] [answered]
My group currently is having trouble getting the battery meter to show for any entity, remains stuck on select entity

**Comment by connorjak** [comment, marked as answer]
Can you send a screenshot? Also, selecting the entities works by clicking on them for all besides the LTVs. LTVs will need to be selected with the Outliner gadget

**Comment by DerryLin-Tamu** [comment]
figured it out using outliner, thank you!
